### PR TITLE
[FUSEQE-12417] JBoss repo

### PR DESCRIPTION
**Pull request by mcarlett**

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -216,23 +222,31 @@
 
 	<repositories>
 		<repository>
-			<id>jboss-soa-qa-maven-repository</id>
-			<url>http://dl.bintray.com/jboss-soa-qa/maven/</url>
+			<id>jboss-public-repository-group</id>
+            <name>JBoss Public Repository Group</name>
```
Comment (osmman) [inline pom.xml]: Please can you fix formating?

File: pom.xml [hunk 2]
```diff
@@ -216,23 +222,31 @@
...
+            <url>http://repository.jboss.org/nexus/content/groups/public/</url>
 		</repository>
 	</repositories>
 
 	<pluginRepositories>
 		<pluginRepository>
-			<id>jboss-soa-qa-maven-repository</id>
-			<url>http://dl.bintray.com/jboss-soa-qa/maven/</url>
+			<id>jboss-public-repository-group</id>
+            <name>JBoss Public Repository Group</name>
```
Comment (osmman) [inline pom.xml]: Please can you fix formating?

File: pom.xml (hunk 1 above)
Comment (mcarlett) [reply pom.xml]: done thanks

File: pom.xml (hunk 2 above)
Comment (mcarlett) [reply pom.xml]: done thanks

**Pull request merged**